Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/kubectl.go

```diff
@@ -165,6 +165,19 @@ var shortForms = map[string]string{
 	"svc":    "services",
 }
 
+// Look-up for resource short forms by value
```

Comment:
@juanvallejo, please take a look at this document it has a lot of useful tips for writing idiomatic Go code :)
https://golang.org/doc/effective_go.html.